seaborn/categorical.py

```diff
@@ -1270,7 +1296,10 @@ def draw_bars(self, ax, kws):
 
             # Draw the confidence intervals
             errcolors = [self.errcolor] * len(barpos)
-            self.draw_confints(ax, barpos, self.confint, errcolors)
+            self.draw_confints(ax, barpos,
+                               self.confint,
+                               errcolors,
+                               draw_perp=self.draw_perp)
```

Comment:
Might as well keep things as either all args or all kwargs since this is internal.